deeplinkdispatch-base/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkEntry.kt

```diff
@@ -126,15 +126,19 @@ sealed class DeepLinkEntry(open val uriTemplate: String, open val className: Str
      */
     override fun compareTo(other: DeepLinkEntry): Int {
         return when {
-            this.firstNonConcreteIndex < other.firstNonConcreteIndex -> -1
+            this.firstNonConcreteIndex < 0 &&
```

Comment:
Can you add a comment here explaining this?

Reply:
Added